spec/components/schema/day_breakdowns.yaml

```diff
@@ -0,0 +1,294 @@
+openapi: 3.0.2
+info:
+  title: Day breakdowns schemas
+  version: 1.0.0
+paths: {}
+components:
+  schemas:
+    Option:
+      type: object
+      required:
+        - id
+        - time_slots
+      properties:
+        option_id:
+          type: integer
+          format: int64
+          example: 1231
+        opening_hours:
+          type: array
+          items:
+            $ref: '#/components/schemas/OpeningHour'
+        time_slots:
+          type: array
+          items:
+            $ref: '#/components/schemas/TimeSlot'
+    OpeningHour:
+      type: object
+      required:
+        - from
+        - to
+      properties:
+        from:
+          type: string
+          example: "14:00"
+        to:
+          type: string
+          example: "18:00"
+    TimeSlot:
+      type: object
+      required:
+        - date_time
+        - vacancies
+      properties:
+        date_time:
+          type: string
+          format: date-time
+          description: Tour-option conduction date, it will be midnight of the conduction date if the activity uses opening hours.
+        vacancies:
+          type: integer
+          example: 10
+        unavailable_reason:
+          $ref: '#/components/schemas/UnavailableReason'
+        price_breakdown:
+          $ref: '#/components/schemas/PriceBreakdown'
+
+    PriceBreakdown:
+      type: object
+      required:
+        - customer_total
+      properties:
+        individuals:
+          type: array
+          items:
+            $ref: '#/components/schemas/IndividualCategoryPriceBreakdown'
+        groups:
+          type: array
+          items:
+            $ref: '#/components/schemas/GroupPriceBreakdown'
+        add-ons:
```

Comment:
Do you want to be using both dashes and underscores in the property names?

Reply:
Fixed, thanks for pointing it out!